apps/coscrad-frontend/src/components/logout-button/logout-button.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import { MemoryRouter } from 'react-router-dom';
+import { hexToRgb } from '../../utils/math/colors';
+import { renderWithProviders } from '../../utils/test-utils';
+import { getDummyConfigurableContent } from '../../utils/test-utils/get-dummy-configurable-content';
+import LogoutButton from './logout-button';
+
+describe('loginbutton', () => {
+    it('should apply the custom backgroundColor', () => {
+        const dummyColor = '#3440eb';
+
+        const expectedColor = hexToRgb(dummyColor);
+
+        const dummyConfigurableContent = getDummyConfigurableContent({
+            themeOverrides: {
+                palette: {
+                    primary: {
+                        main: dummyColor,
+                    },
+                },
+            },
+        });
+
+        renderWithProviders(
+            <MemoryRouter>
+                <LogoutButton />
+            </MemoryRouter>,
+            {
+                contentConfig: dummyConfigurableContent,
+            }
+        );
+
+        const LogoutButtonEl = document.querySelector(`[data-testid="app-bar"]`);
```

Comment:
Why are you searching for the app-bar here? Shouldn't you put a test ID on the `LogoutButton` instead?